Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -36,14 +41,16 @@
 static IWLazyClassLookup s_lazyUIFont("UIFont");
 static const float c_defaultSystemFontSize = 15.0f;
 
-static ComPtr<IDWriteFactory> CreateDWriteFactoryInstance() {
+// Private helper for creating a DWriteFactory
+static ComPtr<IDWriteFactory> __CreateDWriteFactoryInstance() {
```

Comment:
makes sense to follow this convention. May be this should be added to our coding style guided after formal approval from the team

---
In reply to: [78114123](https://github.com/Microsoft/WinObjC/pull/987#discussion_r78114123) [](ancestors = 78114123)